File: myCTCA Mobile/iOS/src/myctca/Models/CareTeam.swift
//
//  CareTeam.swift
//  myctca
//
//  Created by Tomack, Barry on 1/15/18.
//  Copyright © 2018 CTCA. All rights reserved.
//

import Foundation

struct CareTeam: Codable {
    
    var systemId: String = ""
    var name: String = ""
    var userName: String? = ""
    var environmentId: String? = ""
    
    enum CodingKeys: String, CodingKey {
        case systemId, name
        case userName, environmentId
    }
}

File: myCTCA Mobile/Android/src/myctca/app/src/main/java/com/myctca/model/CareTeam.java
package com.myctca.model;

import java.io.Serializable;

/**
 * Created by tomackb on 1/15/18.
 */

public class CareTeam implements Serializable {

    private String systemId;
    private String name;
    private String userName;
    private String environmentId;

    public String getName() {
        return name;
    }

    public String getSystemId() {
        return systemId;
    }

    public String getUserName() {
        return userName;
    }

    public String getEnvironmentId() {
        return environmentId;
    }
}
